app/models/notification.rb

```diff
@@ -1,5 +1,7 @@
 class Notification < ActiveRecord::Base
 #  serialize :departments, Array
+  has_paper_trail
```

Comment:
https://github.com/ReadyResponder/ReadyResponder/blob/development/Gemfile#L32
It seems to be there in the repo...